homeassistant/components/rest_command/__init__.py

```diff
@@ -154,7 +154,7 @@ async def async_service_handler(service: ServiceCall) -> None:
         # register services
         hass.services.async_register(DOMAIN, name, async_service_handler)
 
-    for command, command_config in config[DOMAIN].items():
-        async_register_rest_command(command, command_config)
+    for name, command_config in config[DOMAIN].items():
+        async_register_rest_command(name, command_config)
```

Comment:
This PR is so small... how do you feel about adjusting line 62 to add type hints?
```python
def async_register_rest_command(name: str, command_config: ConfigType) -> None:
```

Reply:
I left open for 24hrs, but it wasn't compulsory...
You can do it in a follow-up PR